### PR TITLE
fix: respect `.` in api versions (fix adsense API)

### LINKED ISSUE #1531: Invalid parameter v1.4 for adsense
Hey there!

When I try to create an instance of adsense API in a typescript file, only version "v1_4" can be passed. But the correct string is "v1.4" because these is the only one key declared in [VERSIONS variable](https://github.com/googleapis/google-api-nodejs-client/blob/master/src/apis/adsense/index.ts#L20).

So, when I execute `const adsense = google.adsense('v1_4')` the following error is returned:
```
.../node_modules/googleapis-common/build/src/apiIndex.js:36
        throw new Error(`Unable to load endpoint ${api}("${version}"): ${e.message}`);
        ^

Error: Unable to load endpoint adsense("v1_4"): ctr is not a constructor
    at Object.getAPI (.../node_modules/googleapis-common/build/src/apiIndex.js:36:15)
```

#### Environment details

  - Node.js version: 11.0.0
  - npm version: 6.4.1
  - `googleapis` version: 36.0.0

#### Steps to reproduce

  1. Install googleapis
  2. Write in a file.ts `const adsense = google.adsense('v1_4')`
  3. Compile with tsc file.ts
  4. Execute with node the generated file.js

I fixed it, renaming the version "v1_4" by "v1.4" in [this line](https://github.com/googleapis/google-api-nodejs-client/blob/master/src/apis/adsense/index.ts#L23) 

That's all, 

Thanks!

### PR DESCRIPTION
**Pull request by AVaksman**
Fixes #1531

- [x] Tests and linter pass
- [x] Code coverage does not decrease (if any source code was changed)

### CODE REVIEW

File: src/apis/adsense/index.ts [hunk 1]
```diff
@@ -20,10 +20,10 @@ export const VERSIONS = {
   'v1.4': adsense_v1_4.Adsense,
 };
 
-export function adsense(version: 'v1_4'): adsense_v1_4.Adsense;
```
Comment (JustinBeckwith) [inline src/apis/adsense/index.ts]: This is a generated file!  See `/*! THIS FILE IS AUTO-GENERATED */` :) This looks to be a bug with the generator.

Comment (AVaksman) [reply src/apis/adsense/index.ts]: On it! Thanks

File: test/test.adsense.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+describe('adsense:1.4', () => {
+  let localAdsense: adsense_v1_4.Adsense, remoteAdsense: adsense_v1_4.Adsense;
+
+  before(async () => {
+    remoteAdsense = await Utils.loadApi<adsense_v1_4.Adsense>(
+        googleapis, 'adsense', 'v1.4');
+  });
+
+  beforeEach(() => {
+    localAdsense = googleapis.adsense('v1.4');
```
[the remark this answers is not in the document]
Comment (AVaksman) [reply test/test.adsense.ts]: Done
Thanks

**Comment by AVaksman** [comment]
I can't see details on why the test failed.

**Pull request merged**